R/package.r

```diff
@@ -133,7 +133,7 @@ package_topics <- function(path = ".", package = "pkgdown") {
   funs <- purrr::map(rd, topic_funs)
 
   tibble::tibble(
-    name = names,
+    name = fs::path_ext_remove(file_out),
```

Comment:
I meant rather than changing this field, you should be changing the code that uses `name` to instead use `file_out`. Or is this data structure just passed to downlit?

Reply:
topics goes to pkg via https://github.com/r-lib/pkgdown/blob/95af5d4a327e90eb0450b8d7f672c6c4c94882a0/R/package.r#L64
what's passed to downlit is https://github.com/r-lib/pkgdown/blob/e1e46f2e13c7efdb17bc4eae40bc3779c2b0b517/R/context.R#L13, I could change that instead :thinking:

Reply:
this is what the next commit does but it does not work, actually. (testing on usethis locally)